### ISSUE TITLE
Frame Tile Background / Border

### ISSUE THREAD

**Issue by fantasai** [raboof/notion#297]
Hello! I just upgraded my OS, and Notion (3-20150613 to 4.0.1) along with it. :) First off, thanks for maintaining this window manager to which I am hopelessly addicted. There were a couple cosmetic things I haven't gotten to work again yet, though:

- trayion dock doesn't resize correctly, possibly due to https://github.com/raboof/notion/issues/268#issuecomment-716943754
- I use xsetbg draw a desktop background, which would show through the transparent frame background, but it's opaque black now no matter what I put in look.cfg. Can't turn it red, and can't turn it transparent.
- I used to have thin borders painted around each tile so I could see the window layout, but this also isn't drawing at all. No matter which border/padding settings I tweak on "frame" or "frame-tiled", whether color or width adjustments, nothing draws differently. (I can adjust the "tab" border drawing no problem, though.)

I saw that you updated the docs for a few other changes (<3) but nothing in this area, so I'm guessing it's a bug.

**Comment by fantasai** [comment]
Ah, forgot to mention, I was using look_cleanviolet.lua

**Comment by fantasai** [comment]
And the fix seems to be reordering the file as follows:

1. base and tab styles
2. dopath("lookcommon_clean")
3. frame styles

I have no idea why.

**Comment by raboof** [comment]
> thanks for maintaining this window manager to which I am hopelessly addicted

Thanks, same here :D

> And the fix seems to be reordering the file as follows

Does that fix both the backgroudn and the thin borders? We had to reimplement some of the look loading/hierarchy logic (#197), and it is indeed possible that it behaves slightly different now. Is there anything that needs to change to the default looks?

**Comment by fantasai** [comment]
@raboof Yes, it fixes both. I don't know why. There's something about how you cascade the styles that's different than before, and I haven't figured out what the differences are or why this results in a fix. Probably all the look.cfg files that use dopath() will need adjustment as a result?

**Comment by wilhelmy** [comment]
@fantasai is there any visible difference in notion's stderr output?
